core/controllers/editor.py

```diff
@@ -120,6 +120,9 @@ def get(self, exploration_id):
                 iframe_restriction=None)
             return
 
+        exploration_status = exp_services.get_exploration_summary_by_id(
```

Comment:
Can we get this from exploration_rights, which we already retrieve below? It would save a datastore GET operation.

Reply:
Yes, that's an overhead.